<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainSettings</class>
 <widget class="QWidget" name="MainSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QXKB5</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabAbout">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="aboutIcon">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="aboutInfo">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabXkb">
      <attribute name="title">
       <string>Xkb</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelSystem">
         <property name="text">
          <string>system info:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="systemInfo">
         <property name="text">
          <string>info</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxStartup">
         <property name="text">
          <string>startup</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditStartup">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>setxkbmap -layout &quot;us,ru(winkeys)&quot; -option &quot;&quot; -option grp:caps_toggle,grp_led:scroll</string>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>86</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Icon</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelBackground">
           <property name="text">
            <string>background color:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditBackgroundColor">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>#191970</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSelColor1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="labelText">
           <property name="text">
            <string>text color:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditTextColor">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>#FFFFFF</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSelColor2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="labelFont">
           <property name="text">
            <string>label font:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditFont">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Arial, 16</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSelFont">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>130</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCache">
      <attribute name="title">
       <string>Cache</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeWidget" name="treeWidgetCache">
         <column>
          <property name="text">
           <string>class</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>layout</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>MainSettings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStartup</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditStartup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditFont</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainSettings</receiver>
   <slot>initXkbLayoutIcons()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditBackgroundColor</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainSettings</receiver>
   <slot>initXkbLayoutIcons()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSelColor1</sender>
   <signal>clicked()</signal>
   <receiver>MainSettings</receiver>
   <slot>selectBackgroundColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSelColor2</sender>
   <signal>clicked()</signal>
   <receiver>MainSettings</receiver>
   <slot>selectTextColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSelFont</sender>
   <signal>clicked()</signal>
   <receiver>MainSettings</receiver>
   <slot>selectFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>initXkbLayoutIcons()</slot>
  <slot>selectBackgroundColor()</slot>
  <slot>selectTextColor()</slot>
  <slot>selectFont()</slot>
 </slots>
</ui>
